dashboard/dashboard.html

```diff
@@ -113,60 +113,72 @@
 			<div class="tab-content clearfix">
 				<div class="tab-pane active" id="energy-score">
 		          	<div class="row border-bottom">
-		          		<div class="col-xs-4"><span class="circle-chart" id="circle-chart"></span></div>
-			          	<div class="col-xs-8">
+		          		<div class="col-xs-3">
+		          			<div class="row center">
+		          				<span class="circle-chart" id="energy-star-score-radial"></span>
+		          				<span class="ranking-num" id="building-energy-star-score">43</span><br><span class="ranking-units">out of 100</span>
+		          			</div>
+		          		</div>
+			          	<div class="col-xs-7">
 			          		<div class="ranking-text">
-			          		In 2015, Hotel Zetta earned an ENERGY STAR score of 43. The hotel median score is 50.
+			          		In 2015, <span id="building-name">Hotel Zetta earned an ENERGY STAR score of <span id="building-energy-star-score">43</span>. The <span id="building-type">hotel</span> median score is <span id="median-energy-star-score">50</span>.
```

Comment:
Totally @thfield, I changed them and lumped all of the classes together in the list above. I'll recheck this evening to make sure I included all of the repeats.